keyboards/gergo/rules.mk

```diff
@@ -17,5 +17,6 @@ CUSTOM_MATRIX 		= yes
 EXTRAKEY_ENABLE		= yes
 CONSOLE_ENABLE  	= yes
 COMMAND_ENABLE		= yes
+BOOTMAGIC_ENABLE = lite
```

Comment:
I can't tag the line, since it wasn't changed.....

But this line:
```make
SRC += matrix.c i2c_master.c
```
should be changed to: 
```make
SRC += matrix.c 
QUANTUM_LIB_SRC += i2c_master.c
```

This is so that if LTO is enabled, it doesn't cause timing issues.

Reply:
Should this be applied to Georgi as well?

Reply:
yeah.